fold/seqfold.go

```diff
@@ -0,0 +1,173 @@
+package fold
+
+import (
+	"fmt"
+	"math"
+	"strings"
+
+	"github.com/TimothyStiles/poly/checks"
+)
+
+// MultibranchEnergies holds the a, b, c, d in a linear multi-branch energy
+// change function.
+// Inferred from:
+// Supplemental Material: Annu.Rev.Biophs.Biomol.Struct.33:415-40
+// doi: 10.1146/annurev.biophys.32.110601.141800
+// The Thermodynamics of DNA Structural Motifs, SantaLucia and Hicks, 2004
+type MultibranchEnergies struct {
+	A, B, C, D float64
```

Comment:
what does each letter individually represent?

Reply:
looking at the section cited and there's actually a fifth parameter not used?

* A - number of helices in the loop
* B - number of unpaired nucleotides in the loop
* C - coxial stacking in the loop
* D - terminal mismatch contributions
* E - base composition of the unpaired nucleotides (probably neglible?)